sherpa/astro/ui/tests/test_astro_ui_utils_simulation.py

```diff
@@ -65,20 +65,23 @@ def test_fake_pha_missing_rmf(idval, clean_astro_ui, tmp_path):
 
     rmf = tmp_path / 'rmf'
 
+    # the error message depends on the backend:
+    # - pyfits: file '{arf}' not found
+    # - crates: unable to open ...: File ... does not exist.
+    #
```

Comment:
This comment is now wrong.

Also, we can simplify the test since we can have a single `pytest.raises` line and just change the match argument.